Languages/Python/answers.json

```diff
@@ -9,8 +9,8 @@
       "details": "Here you would speak on more information for the incorrect answer you have referenced, for more information look at the .json in the android folder."
     },
     {
-      "answer": "It is an example of how you should right the answer, keep it short.",
-      "details": "Here you would speak on more information for the incorrect answer you have referenced, for more information look at the .json in the android folder."
+      "answer": "No",
```

Comment:
@nyrkln No is not an answer. Please input a short answer int "shortAns":

Please review Languages/Android/answers.json